rfc8446.xml

```diff
@@ -4256,10 +4775,27 @@ reject 0-RTT as long as any portion of their recording window overlaps
 the startup time. Otherwise, they run the risk of accepting
 replays which were originally sent during that period.</t>
 
-<t>Note: If the client’s clock is running much faster than the server’s
+<t>Note: If the client's clock is running much faster than the server's,
 then a ClientHello may be received that is outside the window in the
 future, in which case it might be accepted for 1-RTT, causing a client retry,
-and then acceptable later for 0-RTT. This is another variant of
+and then acceptable later for 0-RTT.
+
+<!-- [rfced] Section 8.2:  We had trouble parsing this sentence.
+Please clarify.
+
+Original:
+ Note: If the client's clock is running much faster than the server's
+ then a ClientHello may be received that is outside the window in the
+ future, in which case it might be accepted for 1-RTT, causing a
+ client retry, and then acceptable later for 0-RTT.
+
+Possibly:
+ Note: If the client's clock is running much faster than the server's,
+ then a ClientHello may be received that is outside the window in the
+ future, in which case it might be accepted for 1-RTT - causing a
+ client retry - and then might be acceptable later for 0-RTT. -->
```

Comment:
We weren't sure what "and then acceptable later" means in this context.  If this will be clear to other readers, we're happy to withdraw this question.